php/libraries/NDB_Menu_Filter.class.inc

```diff
@@ -734,7 +734,51 @@ class NDB_Menu_Filter extends NDB_Menu
         $list['data']    = $this->_getFullList();
         $list['headers'] = $this->headers;
 
-        return Utility::arrayToCSV($list);
+        return $this->arrayToCSV($list);
+    }
+
+    /**
+     * Cleans a string to make it safe for variables and such
+     *
+     * @param string $string the string to clean
+     *
+     * @return string The string cleaned up enough to be used as a
+     *                variable name
+     */
+    function getCleanString(string $string): string
+    {
+        $string = trim($string);
+        $string = str_replace('  ', ' ', $string);
+        $string = str_replace(' ', '_', $string);
+        $string = str_replace("'", '', $string);
+        $string = str_replace('"', '', $string);
+        $string = preg_replace('/[^A-Za-z0-9_\/-]*/', '', $string);
+        $string = strtolower($string);
+        return $string;
+    }
+
+    /**
+    * Coverts array into CSV string
+    *
+    * @param array $array the array to be converted
+    *
+    * @return string CSV string of data
+    */
+    function arrayToCSV(array $array): string
+    {
+        $fp = fopen("php://temp", 'w+');
```

Comment:
Is is necessary to do a try/catch or not necessary because it is on temp?

Reply:
I'm not sure I didn't write this code. This PR is just moving it from one place to another.

Reply:
Unless the server run out of memory, there should not be a problem. Even in that case, even a try/catch would not make much difference.